bson/src/main/org/bson/json/JsonReader.java

```diff
@@ -1249,5 +1248,25 @@ protected BsonContextType getContextType() {
             return super.getContextType();
         }
     }
+
+    private static byte[] decodeHex(final String hex) {
```

Comment:
I didn't bother to break this out into a helper since it isn't currently needed anywhere else but here, but could do if you find it unclean.